examples/ssd_mobilenet/MobileNet.js

```diff
@@ -0,0 +1,215 @@
+let nnPolyfill, nnNative;
+if (navigator.ml.isPolyfill) {
+  nnNative = null;
+  nnPolyfill = navigator.ml.getNeuralNetworkContext();
+} else {
+  nnNative = navigator.ml.getNeuralNetworkContext();
+  nnPolyfill = navigator.ml_polyfill.getNeuralNetworkContext();
+}
+
+class MobileNet {
```

Comment:
I would suggest to rename the class name and file name to "SsdMobileNet".

Reply:
I updated my code.